packages/core/src/tenants/SystemContext.ts

```diff
@@ -1,8 +1,17 @@
-import type { StorageProviderData } from '@logto/schemas';
-import { storageProviderDataGuard, StorageProviderKey, Systems } from '@logto/schemas';
+import {
```

Comment:
Not sure about the current test status of this file. Is there any test case may cover this?

Reply:
UTs added